CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ### Changed
 - **cf-forms:** [PATCH] Add transform to fix radio button Firefox rendering bug.
 - **cf-forms:** [MINOR] Add multi-line support to checkboxes/radio buttons.
+- **cf-forms:** [PATCH] Move autoprefixer ignored rule to its own block.
```

Comment:
You can leave it if you want, but seeing as the thing we're correcting hasn't been released yet, this changelog update may be unnecessary?

Reply:
Eh I'll leave it. Doesn't hurt to single it out since it's in its own PR.